driver-sync/src/test/functional/com/mongodb/client/ClientEncryptionCustomEndpointTest.java

```diff
@@ -237,6 +240,19 @@ public void testCustomEndpoint() {
                         + "  \"endpoint\": \"example.com:443\"\n"
                         + "}"),
                 false, MongoClientException.class, MongoCryptException.class, "Invalid KMS response"});
+        data.add(new Object[]{"10. [kmip] default endpoint",
```

Comment:
```suggestion
        data.add(new Object[]{"10. [kmip] endpoint from KMS providers",
```

There is no default endpoint. It must be from KMS providers or part of the DataKeyOptions.

Reply:
Done